http/http-advanced/src/main/resources/application.properties

```diff
@@ -53,9 +53,9 @@ quarkus.keycloak.policy-enforcer.paths.health.enforcement-mode=DISABLED
 quarkus.keycloak.policy-enforcer.paths.version.path=/api/httpVersion/*
 quarkus.keycloak.policy-enforcer.paths.version.enforcement-mode=DISABLED
 quarkus.keycloak.policy-enforcer.paths.hello.path=/api/hello/*
-quarkus.keycloak.policy-enforcer.paths.hello.enforcement-mode=DISABLED        
+quarkus.keycloak.policy-enforcer.paths.hello.enforcement-mode=DISABLED
```

Comment:
Could you remove the formatting changes on the lines that have not changed?

Reply:
This was not an automatic format change, there were space characters here which is wrong.